aries-site/src/pages/foundation/color.js

```diff
@@ -34,14 +41,23 @@ const Color = () => (
       </Box>
     </ContentSection>
     <ContentSection>
-      <Subsection name="Primary" headingSize="small">
-        <Text>
+      <Subsection name="Color Palettes">
+        <BodyText>
+          Color is a key way we express our brand. Using color on interface
+          elements, how typography is presented, displaying data visualizations,
+          and adjusting context of the experience to light and dark are just
+          some of the ways we illustrate our brand.
+        </BodyText>
+        <AnchorCallToAction label="Color Principles" />
```

Comment:
When tabbing through, this Anchor is unreachable? Is that the behavior you get as well?

Reply:
You're right. Let me look into why that is.

Reply:
The reason it is unreachable right now is because it has no href. Once we add an href to it, it becomes part of the tab flow.